src/sentry/incidents/models/alert_rule_activations.py

```diff
@@ -73,6 +73,8 @@ class AlertRuleActivations(Model):
     query_subscription = FlexibleForeignKey(
         "sentry.QuerySubscription", null=True, on_delete=models.SET_NULL
     )
+    # short string descriptor of the specific activation condition met to create the instance (eg. "Release xyz created")
+    activation_reason = models.CharField(max_length=100)
```

Comment:
is the plan for this to be free form text about the reason or an `enum`? (feels like an enum might be nice, just a little nervous about the 100 character limit for a description reason)

Reply:
Yeah, if these are just fixed descriptions it might be nice to use an enum and have the FE write the description

Reply:
I forget if there are any rows in this table already? If so, this column needs to be nullable

Reply:
I was debating how to utilize this 🤔 

The reason here would look like `Release {release.version} created` or `Deploy {deploy.version} created`

so it's not quite an enum.
potentially, could add the 'activation condition' ('release created') and the 'activation key'(`{release.version}`)

so the condition could be an enum - whereas the key would still be a string.

This will only be used for the UI though so figured a string is good enough?

Reply:
@wedamija does it need to be nullable if we're filling in the default in the migration?

Reply:
No, if you have a default it's fine